src/seeders/slots/InSlotSeeder.js

```diff
@@ -78,29 +75,26 @@ class InSlotSeeder extends SlotSeeder {
 
   static generate(data = {}) {
     const slotData = super.generate(data)
-    slotData.dataType = data.dataType || faker.random.arrayElement(Object.values(InSlot.DATA_TYPES))
+    slotData.dataType = data.dataType || faker.random
+      .arrayElement([
+        FloatInSlot.DATA_TYPE,
+        IntegerInSlot.DATA_TYPE,
+        BooleanInSlot.DATA_TYPE,
+        OneOfInSlot.DATA_TYPE,
+      ])
 
     /* this method will delegate and attempt to build fake data based
      * off of what the user has provided. e.g. if the user provided a
      * max value, it will try to give sensible min and defaultValues.
      * the reason all of these values are generated at once is because
      * they are all, per the JOI SCHEMA, related
      */
```

Comment:
this comment seems to have gotten estranged from the method it was providing info for

Reply:
removed it